shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.js

```diff
@@ -31,128 +31,163 @@ exports.fileCourtIssuedDocketEntryInteractor = async (
+    .getPersistenceGateway()
+    .getUserById({ applicationContext, userId: authorizedUser.userId });
 
-  const docketEntryEntity = new DocketEntry(
-    {
-      ...omit(docketEntry, 'filedBy'),
-      attachments: documentMeta.attachments,
-      date: documentMeta.date,
-      documentTitle: documentMeta.generatedDocumentTitle,
-      documentType: documentMeta.documentType,
-      draftOrderState: null,
-      editState: JSON.stringify(documentMeta),
-      eventCode: documentMeta.eventCode,
-      filingDate: documentMeta.filingDate,
-      freeText: documentMeta.freeText,
-      isDraft: false,
-      isFileAttached: true,
-      isOnDocketRecord: true,
-      judge: documentMeta.judge,
-      numberOfPages,
-      scenario: documentMeta.scenario,
-      serviceStamp: documentMeta.serviceStamp,
-      trialLocation: documentMeta.trialLocation,
-      userId: user.userId,
-    },
-    { applicationContext },
-  );
+  const isUnservable = UNSERVABLE_EVENT_CODES.includes(documentMeta.eventCode);
 
-  const workItem = new WorkItem(
-    {
-      assigneeId: null,
-      assigneeName: null,
-      associatedJudge: caseToUpdate.associatedJudge,
-      caseStatus: caseToUpdate.status,
-      caseTitle: Case.getCaseTitle(caseEntity.caseCaption),
-      docketEntry: {
-        ...docketEntryEntity.toRawObject(),
-        createdAt: docketEntryEntity.createdAt,
-      },
-      docketNumber: caseToUpdate.docketNumber,
-      docketNumberWithSuffix: caseToUpdate.docketNumberWithSuffix,
-      hideFromPendingMessages: true,
-      inProgress: true,
-      section: DOCKET_SECTION,
-      sentBy: user.name,
-      sentByUserId: user.userId,
-    },
-    { applicationContext },
+  await Promise.all(
+    docketNumbers.map(async docketNumber => {
+      const caseToUpdate = await applicationContext
+        .getPersistenceGateway()
+        .getCaseByDocketNumber({
+          applicationContext,
+          docketNumber,
+        });
+
+      let caseEntity = new Case(caseToUpdate, { applicationContext });
+
+      const docketEntryEntity = new DocketEntry(
+        {
+          ...omit(subjectDocketEntry, 'filedBy'),
+          attachments: documentMeta.attachments,
+          date: documentMeta.date,
+          documentTitle: documentMeta.generatedDocumentTitle,
+          documentType: documentMeta.documentType,
+          draftOrderState: null,
+          editState: JSON.stringify(documentMeta),
+          eventCode: documentMeta.eventCode,
+          filingDate: documentMeta.filingDate,
+          freeText: documentMeta.freeText,
+          isDraft: false,
+          isFileAttached: true,
+          isOnDocketRecord: true,
+          judge: documentMeta.judge,
+          numberOfPages,
+          scenario: documentMeta.scenario,
+          serviceStamp: documentMeta.serviceStamp,
+          trialLocation: documentMeta.trialLocation,
+          userId: user.userId,
+        },
+        { applicationContext },
+      );
```

Comment:
You'll need to add `docketNumber: caseEntity.docketNumber`.  Same for inside the `editState`.  I.e....


```javascript
editState: JSON.stringify({
    ...documentMeta,
    docketNumber: caseEntity.docketNumber,
}),
```

You may even need to override `docketEntryId` in the `editState`.  @TomElliottFlexion, do you know why overrode `docketEntryId` in `editState` in our interactor?

Reply:
Good catch

Reply:
No, I'm not sure, it seems kind of redundant now?

Reply:
It should be the same as the `documentMeta.docketEntryId`. Only the `docketNumber` should need to be overwritten

Reply:
I want to say we moved away from using things from documentMeta once we'd evaluated it once already.